### DISCUSSION TITLE
New User's Question about how to use Search

### DISCUSSION THREAD

**Discussion by e-patrick** [immich-app/immich#6656] [Q&A]
Hi everybody, I just moved moved my ~150k assets library over to immich and cannot even tell how much I love the app - it's fast, responsive, cross-device, FOSS... What else can one expect? 

But being new to immich there are some things that yet aren't clear to me, especially when it comes to search. Is there any documentation that goes beyond what is found on https://immich.app/docs/features/search?

Or maybe someone more experienced can quickly answer the following questions to get me started:

 * Is there a way to list all the photos that have _not_ been added to an album yet? I used to use this to work through the newly uploaded images to dump those that aren't worth keeping, and eventually add all the others to an album (even if it would only be a dummy album named "Processed" or so)
 ** Is there a different or better way to maintain some sort of Inbox for newly added photos that still need to be reviewed? If, for example, the mobile app could add a tag or specific album on uploading, that would fullfill the same purpose.
 * What is that "meta data" I can search for by prefixing the the search term with "m:"? Are these the attributes from the exif header or anything else?

Thanks in advance, 
Patrick

**Comment by alextran1502** [comment]
Hi Patrick, we are working on a search revamp, which will potentially open doors for the additional mechanism you are asking for.

1. There is no way to filter for this yet.
2. Metadata search includes EXIF data, however, it is currently not working due to the removal of Typesense a few releases back.

**Reply by e-patrick** [reply]
Hi Alex, thanks for the quick response!
I'll wait for the all new search then :)
